Content.Shared/Stealth/SharedStealthSystem.cs

```diff
@@ -22,6 +25,7 @@ public override void Initialize()
         SubscribeLocalEvent<StealthComponent, ComponentInit>(OnInit);
         SubscribeLocalEvent<StealthComponent, ExamineAttemptEvent>(OnExamineAttempt);
         SubscribeLocalEvent<StealthComponent, ExaminedEvent>(OnExamined);
+		SubscribeLocalEvent<StealthComponent, MobStateChangedEvent>(OnMobStateChanged);
```

Comment:
Looks like tabs have been mixed in where spaces should always be used.

Reply:
done